### ISSUE TITLE
Build CMSSW_10_4_0_mtd2_patch1

### ISSUE THREAD

**Issue by fabiocos** [cms-sw/cmssw#25502]
RELEASE_QUEUE:CMSSW_10_4_X
ARCHITECTURE:slc6_amd64_gcc700,slc7_amd64_gcc700
CMSSW_COMMIT:mtd2_p1
CMSDIST_COMMIT:543b39e9d43e9fdb8575ddbfb00fa5e2ef368649

**Comment by fabiocos** [comment]
+1

CMSSW tag corresponding to CMSSW_10_4_0_mtd2 + #25496

**Comment by fabiocos** [comment]
@smuzaffar a patch does not seem to work

**Comment by smuzaffar** [comment]
I am looking in to it

**Comment by fabiocos** [comment]
@smuzaffar thanks, anyway this does not seem to pick the tag but just the master (as the commit is the latest one there...)

**Comment by smuzaffar** [comment]
@fabiocos , looks like using tag for `CMSSW_COMMIT` does not work. For now I have used the commit pointing to tag `mtd2_p1` . Release is building and cmssw tag for release points correctly to the `mrd2_p1`.

**Comment by smuzaffar** [comment]
@fabiocos why you think that it did not pick the right commit?
mtd2_p1 and CMSSW_10_4_0_mtd2_patch1 points to same commit 
https://github.com/cms-sw/cmssw/compare/mtd2_p1...CMSSW_10_4_0_mtd2_patch1
both show the same history
- https://github.com/cms-sw/cmssw/commits/mtd2_p1
- https://github.com/cms-sw/cmssw/commits/CMSSW_10_4_0_mtd2_patch1

am I missing something here?

**Comment by fabiocos** [comment]
@smuzaffar you are correct, I was mislead by the reference to https://github.com/cms-sw/cmssw/tree/master in the comment above...

Ok thanks for your help, so one has to explicitly pick the commit corresponding to the tag in case this exercise is needed (hopefully very rarely). Not sure I will upload the build, only if it is needed, but I wanted to have it ready in case.

**Comment by fabiocos** [comment]
upload all

**Comment by fabiocos** [comment]
release-notes since CMSSW_10_4_0_mtd2

**Comment by slava77** [comment]
> The release notes are ready: https://github.com/cms-sw/cmssw/releases/tag/CMSSW_10_4_0_mtd2_patch1

this doesn't show any pull requests integrated, while there is a difference in the code.
Is this a problem with the release-notes generation or is there a special procedure here (direct commits pushed manually)?

**Comment by fabiocos** [comment]
@slava77 yes, I had seen it, @smuzaffar may better comment, but I guess this is related to the fact that the build has been done using an unorthodox procedure, on the basis of an annotated tag (this is also why I want to have a branch where to manage things in a proper way).

Anyway if you click on the release comparison hn within the release notes you will see the difference correctly reported (although not what we would like to have).

**Comment by smuzaffar** [comment]
@fabiocos @slava77 , this is because there was not merge commits between mtd2 and mtd2_patch branch. It was done directly with commit. I will update the release notes to mention this commit.

Fixes OOTPU for BTLDigitisation: d74dd1812ca6cb948355c4796b5c39a8500f4ad1

**Comment by smuzaffar** [comment]
release notes are updated now.

**Comment by fabiocos** [comment]
@smuzaffar thank you for updating the release notes
